maintainers/ALL-OWNERS

```diff
@@ -3,22 +3,25 @@
 
 approvers:
 - andfasano
+- ashughorla
 - bfournie
 - derekhiggins
-- digambar15
+- dhellmann
```

Comment:
I think @dhellmann has moved to another project, not sure if this addition is correct. I can see his GitHub handle in some GitHub projects of Metal3.io, but those projects are either forgotten to be updated or archived.

Reply:
It's OK to leave me out of the list, since I'm not really active with the project these days.

Reply:
Isn't it so that this file is auto-generated using a script? I would rather check if any repo still has @dhellmann in the OWNERS file, rather than manually removing it from here.

Reply:
That makes sense. A quick check only showed me still in 1 repo: https://github.com/metal3-io/ironic-agent-image/pull/18

Reply:
> Isn't it so that this file is auto-generated using a script? I

Correct, the list has been updated with the output of the script. I would also support removing people from the corresponding repos if needed and then updating the list in this PR. 



> That makes sense. A quick check only showed me still in 1 repo: [metal3-io/ironic-agent-image#18](https://github.com/metal3-io/ironic-agent-image/pull/18)

Thanks @dhellmann